rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -35,6 +35,10 @@ Requires: %{?scl_prefix}rubygem(git)
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix}rubygem(diffy)
+BuildRequires: %{?scl_prefix}rubygem(git)
```

Comment:
I'd be inclined to agree, but only if the same change was made to the Fedora package too (first).